brainstormforce/astra
# AST-2872 - Block Editor Padding Does not Apply to Custom Layout blocks

### Description

- The block editor paddings in the customizer are not applicable to the custom layout blocks. The paddings can be seen in the editor but no padding in the front.

- How to reproduce:
   - Add a custom layout with a group block and add the display rules
   - Now try to change the block editor padding in the customizer.

- JIRA - https://brainstormforce.atlassian.net/browse/AST-2872

There is no change on the padding in the front but the padding is applied in the editor.
### Screenshots

https://d.pr/v/lwOn17

### Types of changes


- Fix: Block Editor Padding Does not Apply to Custom Layout blocks
- Changed selector for css to work for custom layout group, columns and cover blocks.



### How has this been tested?

- Add a custom layout with a group block and add the display rules
- Now try to change the block editor padding in the customizer.
- Check frontend and customizer preview.

### Checklist:
